services/web/server/src/simcore_service_webserver/studies_access.py

```diff
@@ -271,9 +272,14 @@ async def get_redirection_to_study_page(request: web.Request) -> web.Response:
     if is_anonymous_user:
         log.debug("Auto login for anonymous user %s", user["name"])
         identity = user["email"]
+
         await remember(request, response, identity)
+        assert (await get_session(request))["AIOHTTP_SECURITY"] == identity
+        # NOTE: session is encrypted and stored in a cookie in the session middleware
 
-    raise response
+    # WARNING: do NOT raise this response. From aiohttp 3.7.X, response is rebuild and cookie ignore.
+    # TODO: PC: security with SessionIdentityPolicy, session with EncryptedCookieStorage -> remember() and raise response.
```

Comment:
Why not do it in this PR?

Reply:
Sorry, the TODO message is a bit cryptic ... this is a reminder to create a test example and submit a bug issue for aiohttp. IMO there is a bug when they deal with responses with cookies